modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -374,9 +392,11 @@ protected function actions(array $form, FormStateInterface $form_state) {
     // Hide the actions element if it has no buttons.
     $actions['#access'] = isset($actions['previous']) || isset($actions['next']);
     // Once these two steps are reached, the user can't go back.
-    if (in_array($this->stepId, ['offsite_payment', 'complete'])) {
+    if ($this->stepId == 'complete') {
```

Comment:
People should be able to go back to edit information or change payment gateway. You can in any checkout I've seen, especially single page step ones (Apple, etc.)